src/interaction-design/typography.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Typography
+permalink: /:path/:basename/
+---
+There are several typography elements that can enhance the accessibility of web content. These elements include:
+- Font size, line height, line length
+- Font weight
+- Typeface
+
+You may wonder why we discuss these three elements together, while others are mentioned separately. This is because font size, line height, and line length are interrelated and adjusting one will require reevaluating the balance with the others. They cannot be isolated when considering accessibility. And this time we will look at body text first, and then heading text after.
+
+## Body text
```

Comment:
Is it worth adding `To be added.` here like you have for font weight and typeface?